test/multi_tenancy.sh

```diff
@@ -175,6 +180,69 @@ function test_user_has_proper_access() {
     fi
 }
 
+## This function tests acces to ES index/alias from kibana pod.
+function test_user_has_proper_access() {
+    local user=$1; shift
+    local pw=$1; shift
+    local alias=$1; shift
+
+    local kpod=$( get_running_pod kibana )
+    local eshost=$( get_es_svc es )
+
+    os::log::info See if user $user can read alias /$alias
+    get_test_user_token $user $pw false
+    #local xfuser='-H x-forwarded-user:'"$user"
+    #local xocpns='-H x-ocp-namespace:'"$user_project_list"
+    #local xfroles='-H x-forwarded-roles:project_user'
+    
+    #nrecs=$( curl_es_from_kibana $kpod $eshost "/$alias/_count" $test_token -XGET | get_count_from_json )
+    #expected=${1:-$user_project_num}
+    #if ! os::cmd::expect_success "test $nrecs = $expected" ; then
+    #    os::log::error $user cannot access alias /$alias
+    #    curl_es_from_kibana $kpod $eshost "/$alias/_count" $test_token -XGET | python -mjson.tool
+    #    oc exec -c elasticsearch $espod -- es_acl get --doc=roles
+    #    oc exec -c elasticsearch $espod -- es_acl get --doc=rolesmapping
+    #    exit 1
+    #fi
```

Comment:
Can these commented block be dropped? Or is this a mistake?

Reply:
I believe these are here to dump the access docs for when we dynamically were generating them

Reply:
it was a mistake, uncommented. thanks!